card-app/src/UserCard.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import getUsers from './getUsers';
+
+const useStyles = makeStyles({
+  root: {
+    maxHeight: 1000,
+    maxWidth: 545,
+  },
+});
+
+const UserCard = () => {
+  const [user, setUser] = React.useState({
+    avatar:'loading...',
+    email:'loading...',
+    first_name: 'loading...',
+    last_name:'loading...'
+  });
+
+  const [isLoading, setIsLoading] = React.useState(true);
+
+  React.useEffect(() => {
+    getUsers('3')
+      .then(data => {
+        setUser(data);
+        setIsLoading(false);
+      });
+  }, []);
+
+  const classes = useStyles();
+
```

Comment:
Instead of depending on your default '...loading' values in state, you can use the `isLoading` prop to do an early return for the loading message. 

```js
if (isLoading){
  return <p>...Loading</p> 
}
```

I think that makes things a bit easier to reason about.